<ui version="4.0" >
 <class>Frame_System</class>
 <widget class="QFrame" name="Frame_System" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Frame</string>
  </property>
  <property name="frameShape" >
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow" >
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QFrame" name="frame" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <property name="spacing" >
       <number>14</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>9</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_2" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3" >
         <property name="spacing" >
          <number>0</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="significanceCheckBox" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="text" >
            <string>Significance</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="significanceSpinBox" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="suffix" >
            <string> promile</string>
           </property>
           <property name="minimum" >
            <number>50</number>
           </property>
           <property name="maximum" >
            <number>950</number>
           </property>
           <property name="singleStep" >
            <number>50</number>
           </property>
           <property name="value" >
            <number>350</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>0</number>
         </property>
         <item row="2" column="1" >
          <widget class="QLabel" name="label" >
           <property name="layoutDirection" >
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text" >
            <string> x10^  </string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="2" >
          <widget class="QSpinBox" name="zeroSpinBox" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="minimum" >
            <number>-14</number>
           </property>
           <property name="maximum" >
            <number>0</number>
           </property>
           <property name="value" >
            <number>-10</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLineEdit" name="zeroEdit" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="inputMask" >
            <string>d9D; </string>
           </property>
           <property name="text" >
            <string>1</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QCheckBox" name="zeroCheckBox" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="text" >
            <string>Zero range</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="frame_4" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4" >
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_2" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>New Variable</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>Treshold Managment</string>
           </property>
           <property name="checkable" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="widgetResizable" >
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>494</width>
        <height>268</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <widget class="QFrame" name="varFrame" >
         <property name="frameShape" >
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>significanceCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>significanceSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>87</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel" >
     <x>87</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zeroCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>zeroSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>235</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel" >
     <x>327</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zeroCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>zeroEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>191</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel" >
     <x>191</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
